addons/mail/static/src/js/client_action.js

```diff
@@ -372,6 +428,52 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
         return $(QWeb.render("mail.chat.Sidebar", options));
     },
 
+    renderTabPanes: function() {
```

Comment:
method should be private, start with a _: rename it to _renderTabPanes, and move it to private section. also, add docstring